src/zino/scheduler.py

```diff
@@ -79,13 +79,14 @@ def init_state_for_devices(devices: Sequence[PollDevice]):
 
 
 async def load_and_schedule_polldevs(polldevs_conf: str):
-    new_devices, deleted_devices, changed_devices = load_polldevs(polldevs_conf)
+    new_devices, deleted_devices, changed_devices, defaults = load_polldevs(polldevs_conf)
     deschedule_devices(deleted_devices | changed_devices)
-    schedule_devices(new_devices | changed_devices)
+    default_interval_minutes = defaults.get("interval", DEFAULT_INTERVAL_MINUTES)
+    schedule_devices(new_devices | changed_devices, int(default_interval_minutes))
 
 
-def schedule_devices(devices: Sequence[str]):
-    devices = sorted((state.polldevs[name] for name in devices), key=operator.attrgetter("priority"), reverse=True)
+def schedule_devices(new_devices: Sequence[str], default_interval_minutes: int = DEFAULT_INTERVAL_MINUTES):
```

Comment:
The name of the new parameter `default_interval_minutes`  doesnt really explain what its actually used for, maybe `stagger_interval` is better.

Also, is the parameter renamed back to `new_devices` on purpose? pretty sure this was changed in #333

Reply:
No, not on purpose, I was rebasing and seemingly missed that one